Bricks/UUID/0.1.0.toml

```diff
@@ -0,0 +1,8 @@
+[brick]
+name = "`UUID"
```

Comment:
stray backtick?

Reply:
ooooops!  How that escaped my Sauron-like gaze...